egress/tests/testconfig.py

```diff
@@ -0,0 +1,11 @@
+
+# Configure the test suite from the env variables.
+
+import os
+
+DATABASES = {
+    'user': os.environ.get('EGRESS_TESTDB_USER', 'postgres'),
+    'host': os.environ.get('EGRESS_TESTDB_HOST', 'localhost'),
+    'password': os.environ.get('EGRESS_TESTDB_PASSWORD', None),
```

Comment:
Can you find a way to add these only if the env var is set?
I connect by socket, so having host/port set may cause issues.